src/mats_l1_processing/read_parquet_functions.py

```diff
@@ -211,14 +213,20 @@ def read_ccd_data_in_interval(
     if stop.tzinfo is None:
         stop.replace(tzinfo=timezone.utc)
 
+    filterlist = (
+        (ds.field("EXPDate") >= Timestamp(start))
+        & (ds.field("EXPDate") <= Timestamp(stop))
+    )
+    if filter != None:
+        for variable in filter.keys():
+            filterlist = filterlist & ((ds.field(variable) >= filter[variable][0]) &
+                                       (ds.field(variable) <= filter[variable][1]))
```

Comment:
Line is longer than 80 characters and follows a different line break convention to the rest of the file. I suggest:
```suggestion
            filterlist &= (
                (ds.field(variable) >= filter[variable][0])
                & (ds.field(variable) <= filter[variable][1])
            )
```